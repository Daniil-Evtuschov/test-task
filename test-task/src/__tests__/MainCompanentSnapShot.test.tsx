import React from 'react';
import renderer from 'react-test-renderer';
import MainComponent from '../components/mainComponent/MainCompanent';

test('MainComponent renders correctly', () => {
  const component = renderer.create(<MainComponent icons={['icon1', 'icon2', 'icon3']} />).toJSON();
  expect(component).toMatchSnapshot();
});
