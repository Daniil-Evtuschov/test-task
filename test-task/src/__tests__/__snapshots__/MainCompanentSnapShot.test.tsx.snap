// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainComponent renders correctly 1`] = `
<div
  className="wrap"
>
  <span
    className="icon"
  />
  <span
    className="iconName"
    data-testid="icon-name"
  >
    icon2
  </span>
  <button
    className="iconChangeButton"
    onClick={[Function]}
  >
    next Icon
  </button>
</div>
`;
